examples/teleport-usage/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3.11.1-alpine3.17
+WORKDIR /app
+COPY requirements.txt /app/
+COPY teleport_usage.py /app/
+RUN pip install --upgrade pip setuptools
+RUN pip install -r requirements.txt
+ENTRYPOINT ["python", "/app/teleport_usage.py"]
```

Comment:
I don't know much about Python, so I may be off here, but in order to minimize the number of layers that need to be rebuilt, I would do things in this order:

- pip install --upgrade pip setuptools
- copy requirements.txt /app/
- pip install -r requirements.txt
- copy teleport_usage.py

This means the most common case of the python script changing but the dependencies remaining the same will be a very quick rebuild, as only the top layer with the script needs updating.